docs/source/methods/random_ray.rst

```diff
@@ -754,6 +756,18 @@ behavior if a single simulation cell is able to score to multiple filter mesh
 cells. In the future, the capability to fully support mesh tallies may be added
 to OpenMC, but for now this restriction needs to be respected.
 
+Flux tallies are handled slightly differently than in Monte Carlo. By default,
+in MC, flux tallies are reported in units of tracklength (cm), so must be
+manually normalized by volume by the user to produce an estimate of flux in
+units of cm\ :sup:`-2`\. Alternatively, MC flux tallies can be normalized via a
+separated volume calculation process as discussed in the :ref:`Volume
+Calculation Section<usersguide_volume>`. In random ray, as the volumes are
+computed on-the-fly as part of the transport process, the flux tallies can
+easily be reported either in units of flux (cm\ :sup:`-2`\) or tracklength (cm).
+By default, the unnormalized flux values (units of cm) will be reported. If the
+user wishes to received volume normalized flux tallies, then an option for this
```

Comment:
Yes, that was my thinking. I'm definitely open to switching to reporting volume normalized results by default (as that is probably the more natural way of reporting results for this solver). However, my thought was that people may be commonly comparing against the MGMC results, so may be useful to have it report apples-to-apples numbers by default to avoid confusion. We can always add some warnings through in the docs about this to highlight the difference. Anyway, I'm curious to hear what people's vote for the default should be.

Reply:
I think the way you have done it makes sense, and it's nice there is an option to normalize it.